src/legacy/Routes.tsx

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable */
 
-import React from 'react'
+import React, { useEffect, useState } from 'react'
```

Comment:
Please do not add new components in the legacy code. All the edits of legacy code should be minimal